### PR TITLE
AHOYAPPS-477: Add video app link to readme

### PR DESCRIPTION
**Pull request by timrozum**
Add video app link to readme like https://github.com/twilio/video-quickstart-js.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 [ ![Download](https://img.shields.io/badge/Download-iOS%20SDK-blue.svg) ](https://www.twilio.com/docs/video/ios#add-the-sdk)
 [![Docs](https://img.shields.io/badge/iOS%20Docs-OK-blue.svg)](https://twilio.github.io/twilio-video-ios/docs/latest/index.html)
 
+**NEW:** Please check out the newly open-sourced [multi-party video application](https://github.com/twilio/twilio-video-app-ios)
+built with iOS Video SDK.
```
Comment (timrozum) [inline README.md]: @aaalaniz below is some feedback from @paynerc:
```
I’m not sure we should call it multi-party video application. We already have the MulitPartyExample quick start application that we refer to in the docs. I am wondering if we should call it something else?
```
Just call it `video app`? I copied from JS quickstart.

File: README.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 [ ![Download](https://img.shields.io/badge/Download-iOS%20SDK-blue.svg) ](https://www.twilio.com/docs/video/ios#add-the-sdk)
 [![Docs](https://img.shields.io/badge/iOS%20Docs-OK-blue.svg)](https://twilio.github.io/twilio-video-ios/docs/latest/index.html)
 
+**NEW:** Please check out the newly open-sourced [multi-party video application](https://github.com/twilio/twilio-video-app-ios)
```
Comment (paynerc) [inline README.md]: I’m not sure we should call it `multi-party video application`. We already have the `MulitPartyExample` quick start application that we refer to in the docs. I am wondering if we should call it something else?

Comment (timrozum) [reply README.md]: @aaalaniz call it `video app`? I copied from JS quickstart.

Comment (timrozum) [reply README.md]: Or `Twilio Video iOS App`? That is what the repo title is.

Comment (aaalaniz) [reply README.md]: Let's go with `video collaboration app`

**Pull request merged**